src/checklist.md

```diff
@@ -1,43 +1,72 @@
-# Checklist
-Use this checklist to follow along with the guide. We've made some choices for you in this checklist, but you can substitute other choices if you decide to after reading the guide.
+# Checklists
+These checklists are here to help you follow along with all the recommendations from [our guide](/), with some default choices already made for you. [The first checklist](#getting-safer-today) is for everything you should do today, and [the second](#staying-safe-into-the-future) is to help you stay on track with longer-term habits.
 
-## Your passwords
-- Get 1Password or BitWarden
+# Getting safer today
+These are all the things you can do today to make your digital life safer.
+
+## Passwords
+- Get a password manager like 1Password or BitWarden
 - Turn on 2FA wherever you can
 
-## Your network
+## Your home network
+- Change your router's admin password from the default password
 - Add a WPA3 or WPA2-AES password to your wi-fi network
 - Change your router's DNS servers to 1.1.1.1 and 1.0.0.1
+- Turn off WPS and uPnP (Universal Plug-and-Play)
 
-## Your laptops and desktops
-- Add a password to your login
+## Laptops and desktops
+- Add a password to your computer login
 - Learn how to lock your computer
 - Clean up unused user accounts
 - Install pending OS updates
 - Uninstall anti-malware scanners
 - Set up automatic cloud backups with Backblaze or Carbonite
 - Turn on disk encryption
 
-## Your phones and tablets
+## Phones and tablets
 - Add a long numeric passcode to your lock screen
-- Change your device's name to something without your name in it
-- Install pending OS and app updates
 - Install and use Signal for messaging
+- Install pending OS and app updates
 - Uninstall unused apps
 - Audit remaining app permissions
+- Change your device's name to something without your name in it
 - Remove old Bluetooth device pairings
+- Turn off Bluetooth if you aren't using it
 
 ## Internet of Things (IoT)
 - Throw out your smart speakers
+- Decide which other devices you want to keep
 - Review device privacy settings
 - Disconnect devices from the network if possible
 
-## Your browser
+## Browsing the internet
 - Switch to Firefox
-- Use DuckDuckGo for web searches
 - Install the Privacy Badger and HTTPS Everywhere extensions
+- Switch your browser's default search engine to DuckDuckGo
 
 ## Your services
 - Consolidate your services
-- Check Have I Been Pwned? for your accounts
 - Check your privacy settings in all your services
+- Check Have I Been Pwned? for your accounts
```

Comment:
should we have links from here to Firefox, DDG, HIBP?

Reply:
Added.